### DISCUSSION TITLE
Sakura_embyboss/use_cases/FAQ/

### DISCUSSION THREAD

**Discussion by giscus** [berry8838/Sakura_embyboss#17] [Announcements]
# Sakura_embyboss/use_cases/FAQ/

Emby Bot 指南

http://127.0.0.1:8000/Sakura_embyboss/use_cases/FAQ/

**Comment by berry8838** [comment]
**emmmmmmmm** 这是一条测试信息

**Reply by berry8838** [reply]
如有问题请直接发issue，这里不常看了

**Comment by qingyi241** [comment]


**Reply by berry8838** [reply]
现在呢？

**Comment by qingyi241** [comment]
现在有一台机器搭建的可以正常使用，只是用户到期不会自动禁止，手动有时候可以，有时候也无效。另外其他VPS用脚本搭建这个BOT，还是会出现数据库错误这个提示，导致bot
用不了。感谢了

FisrtLover ***@***.***>于2024年5月20日 周一12:11写道：

**Comment by pattonbina** [comment]
用户绑定后会机器人会在群组发消息，能否支持自动删除绑定消息

**Reply by berry8838** [reply]
没打算
